curriculum/challenges/english/01-responsive-web-design/applied-visual-design/create-texture-by-adding-a-subtle-pattern-as-a-background-image.english.md

```diff
@@ -52,7 +52,7 @@ tests:
 ```html
 <style>
   body {
-    background: url("https://i.imgur.com/MJAkxbh.png");
+    background: url("https://cdn-media-1.freecodecamp.org/imgr/MJAkxbh.png");
```

Comment:
Do we need to change this, I believe we can just enter any URL there, the current one doesnot even make sense as a BG

Reply:
I think it's fine as is.  The URL does have to be an image for the camper to see a change.  Also, this image does create the subtle change that the instructions talk about.

Reply:
@thecodingaviator, thanks for the catch. If it is alright with you, I would rather differ content modifications specially in the curriculum to another PR since it would require a further discussion.

Reply:
Since we're doing it to the Egnlish file, we'll need to do it to the other languages too. If you want to do it in this PR, fine enough, I'm totally fine if you want another file

Reply:
Thank you @thecodingaviator, I took note of suggestions, and I will address them in a separate PR.

Reply:
If everything else other than changing the background look acceptable, please feel free to merge this PR.